Examples/DataRepresentation/Image/Image4.cxx

```diff
@@ -136,7 +138,7 @@ int main(int, char *[])
   //  Software Guide : EndLatex
 
   // Software Guide : BeginCodeSnippet
-  const ImageType::SpacingType& sp = image->GetSpacing();
+  const ImageType::SpacingType & sp = image->GetSpacing();
```

Comment:
Isn't `&` and `*` alignment with type's name? My memory might be wrong.

Reply:
The code is consistent.  I don't know if there was a style.

`PointerAlignment: Middle` --> `PointerAlignment: Left` ?

Reply:
I prefer "Middle"